### PR TITLE
monit: fix UI issues

### PR DESCRIPTION
**Pull request by fbrendel**
fix for #2515

### CODE REVIEW

File: src/opnsense/mvc/app/controllers/OPNsense/Monit/Api/SettingsController.php [hunk 1]
```diff
@@ -310,6 +310,7 @@ public function notificationAction()
                 Config::getInstance()->save();
                 if ($this->mdlMonit->configDirty()) {
                     $result['status'] = 'ok';
```
Comment (fichtner) [inline src/opnsense/mvc/app/controllers/OPNsense/Monit/Api/SettingsController.php]: does it need both ? oO

**Comment by fbrendel** [comment]
I'd say yes.
It's a feedback that all settings were imported correctly.
On the other hand 'No news are good news'... :smirk:

File: src/opnsense/mvc/app/controllers/OPNsense/Monit/Api/SettingsController.php (hunk 1 above)
Comment (fbrendel) [reply src/opnsense/mvc/app/controllers/OPNsense/Monit/Api/SettingsController.php]: I'd say yes.
It's a feedback that all settings were imported correctly.
On the other hand 'No news are good news'... ￼

**Comment by fichtner** [comment]
Merged, thank you!

**Pull request merged**